modules/Bio/EnsEMBL/ArchiveStableId.pm

```diff
@@ -529,5 +535,17 @@ sub successors {
   return $self->{'successors'};
 }
 
+sub meta_value {
+  my $self = shift;
+  $self->{meta_value} = shift if (@_);
+  return $self->{'meta_value'};
+}
+
+sub species_id {
+  my $self = shift;
+  $self->{species_id} = shift if (@_);
+  return $self->{'species_id'};
+}
+
```

Comment:
This is a feature, not an adaptor. Bio::EnsEMBL::Feature does have a species method, but I don't think that class is appropriate for inheritance here.